pkgs/top-level/aliases.nix

```diff
@@ -803,11 +809,13 @@ mapAliases ({
   xlibs = xorg; # added 2015-09
   xpraGtk3 = xpra; # added 2018-09-13
   xv = xxv; # added 2020-02-22
+  yacc = bison; # moved from top-level 2021-03-14
   youtubeDL = youtube-dl;  # added 2014-10-26
   ytop = throw "ytop has been abandoned by upstream. Consider switching to bottom instead";
   yubikey-neo-manager = throw "yubikey-neo-manager has been removed because it was broken. Use yubikey-manager-qt instead."; # added 2021-03-08
   yuzu = yuzu-mainline; # added 2021-01-25
   zdfmediathk = mediathekview; # added 2019-01-19
+
```

Comment:
Why though?

Reply:
The alphabetical sorting ends here. I can drop that if it is a real problem.

Reply:
At least it doesn't seem to match any of the commit messages.

Reply:
Please add a separate commit for this change if you include it.

Reply:
This is really bikeshedding.

Reply:
Alright, a better suggestion: Make one PR per alias. 237 changed files is too much to review.

Reply:
I did one alias per commit which should be easily reviewable.